modules/power-monitoring-metrics-overview.adoc

```diff
@@ -0,0 +1,71 @@
+
+|`kepler_container_cpu_instructions_total` |The total CPU instructions used by the container that utilizes hardware counters. CPU instructions is a metric that accounts how the CPU is used.
+
+|`kepler_container_cache_miss_total` |The total cache miss that occurs for a container that uses hardware counters.
+
+|`kepler_container_cgroupfs_cpu_usage_us_total` |The total CPU time used by a container reading from control group statistics.
+
+|`kepler_container_cgroupfs_memory_usage_bytes_total` |The total memory in bytes used by a container reading from control group statistics.
+
+|`kepler_container_cgroupfs_system_cpu_usage_us_total` |The total CPU time in kernel space used by the container reading from control group statistics.
+
+|`kepler_container_cgroupfs_user_cpu_usage_us_total` |The total CPU time in user space used by a container reading from control group statistics.
+
+|`kepler_container_bpf_net_tx_irq_total` |The total number of packets transmitted to network cards of a container that uses the BPF tracing.
+
+|`kepler_container_bpf_net_rx_irq_total` |The total number of packets received from network cards of a container that uses the BPF tracing.
+
+|`kepler_container_bpf_block_irq_total` |The total number of block I/O calls of a container that uses the BPF tracing.
+
+|`kepler_node_info` |The node metadata, such as the node CPU architecture.
+
+|`kepler_node_core_joules_total` |The total energy consumption across CPU cores used by all containers running on a node and operating system.
+
+|`kepler_node_uncore_joules_total` |The cumulative energy consumption by uncore components used by all containers running on the node and operating system. The number of components might vary depending on the system.
+
+|`kepler_node_dram_joules_total` |The total energy consumption of DRAM by all containers running on the node and operating system.
+
+|`kepler_node_package_joules_total` |The cumulative energy consumed by the CPU socket used by all containers running on the node and operating system. It includes all core and uncore components.
+
+|`kepler_node_other_host_components_joules_total` |The cumulative energy consumption of host components, excluding CPU and DRAM, used by all containers running on the node and operating system. Generally, this metric is the energy consumption of ACPI hosts.
+
+|`kepler_node_platform_joules_total` |The total energy consumption of the host. Generally, this metric is the host energy consumption from Redfish BMC or ACPI.
+
+|`kepler_node_energy_stat` |Multiple metrics from nodes labeled with container resource utilization control group metrics that are used in the model server.
+
+|`kepler_node_accelerator_intel_qat` |The utilization of the accelerator Intel QAT on a certain node. If the system contains Intel QATs, {PM-kepler} can calculate the utilization of the node's QATs through telemetry.
```

Comment:
This list is more in a logical order. First, we mention metrics related to energy consumption, then CPU metrics, control group, ... for the container, and then, in the same order, for the nodes. I hope that explains it, thank you!

Reply:
If that's the case, then there should be categories as you mentioned.  Otherwise, I would also vote to alphabetize this list because the list almost seems random right now.